src/components/custom/signup.tsx

```diff
@@ -47,18 +47,27 @@ const Form = styled.form<FormProps>`
         width: 100%;
     }
 `
-const ResponseWrapper = styled.div`
+const ResponseWrapper = styled(Flex)`
     justify-content: center;
-    max-width: 33rem;
+    max-width: 24rem;
     margin: 0 auto;
     flex-direction: column;
     padding: 2rem 1rem;
+    gap: 12px;
+    margin-top: -100px;
+    @media ${device.mobileL} {
+        max-width: 40rem;
+    }
 `
 
-const ConfirmationMessage = styled.div`
+const ConfirmationMessage = styled(Header)`
     text-align: center;
     font-size: 16px;
     word-wrap: break-word;
+    line-height: 24px;
+    @media ${device.mobileL} {
+        font-size: 14px;
+    }
```

Comment:
Properties `text-align, font-size, line-height` already exist in `Header` via passing `type`. Are we need to reach special view here?